Dockerfile

```diff
@@ -30,13 +30,8 @@ RUN apt-get update --fix-missing && apt-get install -y \
     libxrender1 \
     wget \
     libevent-dev \
-    build-essential && \
-    rm -rf /var/lib/apt/lists/*
-
-# Install Java.
-RUN echo "deb http://http.debian.net/debian jessie-backports main" >>/etc/apt/sources.list
-RUN apt-get update
```

Comment:
I think this update was previously necessary because of the `rm -rf /var/lib/apt/lists/*` immediately proceeding it--so this change should make the build a bit faster too.

Reply:
Pretty sure updating is just necessary after a new source is added.

Reply:
@brendan-ai2 look earlier in the Dockerfile.  We no longer need to add a new source (since we're on stretch) and we no longer need to update here because we do earlier.  An update would have been required for all installs after `rm -rf /var/lib/apt/lists/*`, so the previous iteration of this Dockerfile was inefficient (it would create the apt lists, nuke them, and then recreate them).

Reply:
You can disregard my last comment--we're both right.

Reply:
Right, reorganizing would have been ideal. At any rate, thanks again for fixing this!